.just/maven.just

```diff
@@ -12,8 +12,7 @@ _clean-m2:
     exit 0
 
 # mvn clean
-_clean-maven MVN=default_mvn *FLAGS=default_flags:
-    {{MVN}} clean {{FLAGS}}
+_clean-maven MVN=default_mvn *FLAGS=default_flags: (mvn MVN "clean" "" FLAGS)
```

Comment:
Trying to delegate as many maven related recipes as possible to the mvn recipe, since it handles "skip" commit messages and has good error messages.